libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/offboard/MapboxOffboardRouter.kt

```diff
@@ -70,22 +71,48 @@ class MapboxOffboardRouter(
                     response: Response<DirectionsResponse>
                 ) {
                     directionRequests.remove(requestId)
+                    val url = call.request().url.toUrl()
```

Comment:
For "security" reasons, do we want to filter out the `access_token` from the URL so it's not leaked out in the Logcat?

Reply:
:+1: